### DISCUSSION TITLE
personal-summary/cs/18-计算机科学与技术-DYC-保研-@USTC

### DISCUSSION THREAD

**Discussion by giscus** [Hainanu-Application/Hainanu-Application.github.io#38] [Announcements]
# personal-summary/cs/18-计算机科学与技术-DYC-保研-@USTC

A magical documentation generator.

https://hainanu-application.github.io/#/personal-summary/cs/18-%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%A7%91%E5%AD%A6%E4%B8%8E%E6%8A%80%E6%9C%AF-DYC-%E4%BF%9D%E7%A0%94-@USTC

**Comment by Haodong-Lei-Ray** [comment]
哇，好多保研的都是转专业的
